src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/PerformanceController.php

```diff
@@ -44,12 +44,11 @@ class PerformanceController extends FrameworkBundleAdminController
     /**
      * Displays the Performance main page.
      *
-     * @Template("@PrestaShop/Admin/Configure/AdvancedParameters/performance.html.twig")
      * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))", message="Access denied.")
      *
      * @param FormInterface $form
      *
-     * @return array
+     * @return Response
```

Comment:
can we imagine to force the type hint instead ?

```diff
- public function indexAction(FormInterface $form = null)
+ public function indexAction(FormInterface $form = null) : Response
```

Reply:
I was convinced this was a breaking change, but apparently it's possible!